images/Telemetry/TelemetryEmojii.h

```diff
@@ -0,0 +1,13 @@
252,0341,'C',0305,0311,'S',0231,'?',0374,'c',')',0256,0271,'^',0306,0253,'c','}',0223,0200,'i',02,0231,0300,'j',0314,0324,0201,0354,0304,'i','(',0340,0236,026,'+',0264,'L','k',0324,025,033,0333,0321,'o',0302,0206,':','Y',015,0255,'O',0367,'U','V',022,0366,0242,0271,'V',0306,0201,'e',0250,0305,'8',0206,0326,'=','7',0270,'"',0327,0331,0270,'$',0363,0260,0362,0201,'m',0210,0,'T',0134,0264,0271,'b','L',015,0226,0272,'M',0223,047,'u','4',0306,'M',0203,0337,0334,017,'V','p',0247,',','G',0271,0276,0273,'H','p','$',0177,0244,0377,'m',047,'x','8','C','?',0253,'X','Y',0302,0231,0205,02,'(',01,' ','r',0323,0272,'b',036,'^','s','@','W',0353,'u','E',06,'b',0312,0264,'F',030,0252,'_',',',02,'Q',0226,'x','{',0323,0215,0263,'0','r',0326,0267,0323,0327,' ',0240,'L','m',0320,0257,'p','/','[',0260,'X',0206,'U','y','z',0214,'j','D',0321,'D',0351,0317,0337,'.',0361,0203,0277,0205,016,':','r',022,'9',0376,'X','_','@','q','j',0325,'Q',0301,0267,036,0235,0316,'~',025,'f',0237,0334,0216,0240,'P',0206,0272,'"',07,'9','j','7',' ',0332,0322,0200,'X',0273,'&',0324,030,'5',0371,'l','6',0274,0300,0342,015,0356,'5','!',015,0273,0237,0234,'F',0226,01,'T',0306,0344,'3',0224,0337,012,'t',0312,'D','s','=',0317,0272,0220,'b',',',0330,0314,0270,'J',0352,0313,'/',0227,0356,027,'~','G',0272,0273,'/',0322,0362,'W',0277,'k',0331,'4','K',03,'|',0251,0204,0203,'h',0231,0267,0316,'6',020,0241,0214,0354,0235,0303,0310,0364,0327,0375,'a',0312,'z',0261,0275,'m',0352,0274,0275,031,'`','2','s','W','j',0322,'O',',',0211,0355,0265,0205,0240,0330,0372,'V','&',037,0207,'5',033,0230,'>',0323,027,015,0344,0221,0261,'{',06,0361,'-',0201,07,0271,'X',0211,036,0213,']','`',0340,0310,'h',035,04,0224,025,0245,0323,'Z','"',025,0343,0253,016,0224,'i',0235,0257,014,0257,0315,'b','>',0255,0235,'_',0267,030,021,'S',0373,0223,015,'o',0245,0206,'{','Y',0367,0251,0365,036,026,'e',0234,016,'8',012,'X',0240,'}',0207,'v','q',0,0225,0252,0362,'A',0326,'(',0323,0261,0367,'A','8',0211,0215,0356,0212,0236,0243,0230,0267,0311,0325,'+',0247,'N',0374,0300,'y',0324,0303,0220,'6',0224,'%','I',0,'}',0210,0257,0277,'"',0332,0216,0235,0371,0276,'2',0220,0304,0316,047,'b',033,0354,0134,0354,'|',033,0224,0262,0324,033,'x','p',0266,0271,0355,0224,'A',0273,0366,015,037,'*','6',0321,'B',0252,0311,0352,0306,0336,0233,0342,0222,0345,013,0323,022,0207,022,'9',0224,0230,'x',03,'&',037,',','F',0240,'H',0275,0325,'9',030,0267,0210,0327,'9',0224,0364,'@','9',0237,0270,0356,0244,'M',025,'6','X',',',022,'+',0243,0313,0366,0253,0256,0252,0306,'o',0274,0377,0344,'l','Z',0241,0236,0323,'p','O',0,0256,020,'!','p','z',0255,'L','q',0206,023,0255,'=',0230,05,0310,'(',0264,0200,'*',0366,'*',03,'-','7',0340,'D',0324,'N','i',0317,0214,0226,0261,0275,0312,'J',0306,0236,0306,0134,0214,0265,0366,'>','s','=',027,0300,'(',0274,0300,0322,013,'J',0320,'r','Z',':','I','L',0246,0305,'"','o','}',0323,033,0350,0212,0312,'e',0255,037,0207,']',0231,0327,'d',0260,'c','m',0243,'b','?',03,0200,'[',0320,0376,0310,0311,'M','z',0,0,0,0,'I','E','N','D',0256,'B','`',0202,};
```

Comment:
This seems to be a downscaled version of Apple's "Chart with Upwards Trend (U+1F4C8)" emoji according to Emojipedia. I'm not a lawyer but I'm fairly sure you're not supposed to redistribute it under GPL.